src/cloud_db/cloud_db.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from dotenv import load_dotenv
+import supabase
+from postgrest import APIResponse
+
+from touch_sensor.touch_sensor import Tap
+from utils.touch_sensor_utils import tap_to_db_entry
+
+load_dotenv()
+
+
+class CloudDBClient:
+    """Client for interacting with a cloud database using the Supabase API."""
+
+    def __init__(self):
+        """
+        Initializes the client with database credentials.
+
+        Attributes:
+            client: Supabase Client instance.
+        """
+        url = os.getenv('DATABASE_URL')
```

Comment:
is os.getenv getting environment variables for this and if so, how are the environment variable set?